rewrite-maven/src/main/java/org/openrewrite/maven/MavenVisitor.java

```diff
@@ -283,7 +284,7 @@ public ResolvedDependency findDependency(Xml.Tag tag, @Nullable Scope inClasspat
     }
 
     public MavenMetadata downloadMetadata(String groupId, String artifactId, ExecutionContext ctx) {
```

Comment:
Wondering if the HttpSender should be something that can be configured in the context? Right now, any recipe using this is going to be using the default with no ability to change the implementation (not a big deal).

Reply:
Thought about it, but I don't know how we would serialize that over the wire. So deferring to later.

Reply:
oh good point.